AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -48,6 +48,7 @@ jiji.com,edp24.co.uk,technologyreview.com,inews.co.uk,nordbayern.de,thelocal.ch,
 !
 ! SECTION: Subscriptions - Regular rules
 !
+thegamer.com###newsletter-popup-container
```

Comment:
please add thegamer.com to the existing rule

`gamerant.com,androidpolice.com,xda-developers.com###newsletter-popup-container`

Reply:
done!